content/docs/tasks/telemetry/distributed-tracing/index.md

```diff
@@ -137,6 +137,35 @@ public Response bookReviewsById(@PathParam("productId") int productId,
 
 When you make downstream calls in your applications, make sure to include these headers.
 
+## Trace Sampling
+
+Istio captures a trace for all requests by default. Every time you
+access `/productpage` you see a corresponding trace in the Jaeger
```

Comment:
You need to preface this with something like:

When using the Bookinfo sample application for example,